components/formats-gpl/pom.xml

```diff
@@ -208,7 +208,7 @@
                 <additionalClasspathElement>${basedir}/../../ant/</additionalClasspathElement>
               </additionalClasspathElements>
               <classpathDependencyExcludes>
-                <classpathDependencyExclude>${project.groupId}:mdbtools-java</classpathDependencyExclude>
+                <classpathDependencyExclude>org.openmicroscopy:ome-mdbtools</classpathDependencyExclude>
```

Comment:
A corresponding change should also be done in `components/formats-gpl/build.xml` in the `test-no-mdb` target to exclude the `ome-mdbtools` jar.